sds_data_manager/constructs/ialirt_ingest_lambda_construct.py

```diff
@@ -62,20 +62,33 @@ def create_dynamodb_table(self) -> aws_dynamodb.Table:
             # Restore data to any point in time within the last 35 days.
             # TODO: change to True in production.
             point_in_time_recovery=False,
-            # Partition key (PK) = Mission Elapsed Time (MET).
+            # Partition key (PK) = ingest year (YYYY).
             partition_key=ddb.Attribute(
-                name="met",
+                name="apid",
                 type=ddb.AttributeType.NUMBER,
             ),
-            # Sort key (SK) = Ingest Time (ISO).
+            # Sort key (SK) = Mission Elapsed Time (MET).
             sort_key=ddb.Attribute(
-                name="ingest_time",
-                type=ddb.AttributeType.STRING,
+                name="met",
+                type=ddb.AttributeType.NUMBER,
             ),
             # Define the read and write capacity units.
             # TODO: change to provisioned capacity mode in production.
             billing_mode=ddb.BillingMode.PAY_PER_REQUEST,  # On-Demand capacity mode.
         )
+
+        # Add a GSI for ingest time.
```

Comment:
How important is this GSI? Are these just one-off searches once/year kind of thing, or are we regularly going to be querying for ingest time?

I'm curious if we could simply do the comparisons against the attributes of the main table instead of creating a new index. Our ingest time is also constrained to probably a +/- 10 minute buffer, so the ingest time probably doesn't get you a lot of great additional information to search by and you could do a query against +/- 10 minutes on the main index items even.